CHANGES.md

```diff
@@ -95,6 +95,8 @@ config option `[scheduling]stop after cycle point`.
 
 ### Fixes
 
+[#3917](https://github.com/cylc/cylc-flow/pull/3917) - Fix a bug that caused one of the hostname resolution tests to fail in certain environments.
+
```

Comment:
BTW: We try to keep the changelog focused on user-facing changes (especially as there is going to be a very long list of changes for Cylc8), no probs though.